docs/content/metric-stores/overview.md

```diff
@@ -44,6 +44,31 @@ f_total_cost = Feature(
 )
 ```
 
+Then when the FeatureView that hosts the above feature is being materialized,
+the related metrics will also be reported during the materialization process.
+
+By default, Feathub would only report the metrics directly defined in the
+FeatureView to be materialized. If metrics defined in the source FeatureViews
+should also be reported, please enable the `keep_source_metrics` parameter when
```

Comment:
Can you update the doc to also make it clear that this will be done recursively?